tealeaf/platform/PluginManager.h

```diff
@@ -40,6 +40,7 @@
 - (void) onPause;
```

Comment:
That way, safari and google play store are _other_ apps. These are also apps that trigger our app to be opened